### DISCUSSION TITLE
Pinned Lists + Private Lists

### DISCUSSION THREAD

**Discussion by Zanger67** [community/community#121515] [Other Feature Feedback, Questions, & Ideas]
I've been using Lists to ***sort and categorize repos*** for myself and others but it would be great imo to have a way to either pin a few of the lists on the $\color{yellow}Starred$ tab. 

Namely, I have **"Current Projects," "My Projects," and "Hackathons"** as the main lists that I use for others' reference, while the others are just for sorting and personal interest. I've linked them to my Readme but it would be nice to have a setting there too. What I'm noticing is that when I star new repos for self interest for instance, since the default tends to go to _sort by most recently updated_, it'll be knocked down the list of lists.

Being able to **private lists** would also be great especially for my misc. lists such as "interesting" which are things I want to look into and read into or "Class" which is just full of private repos hence appearing empty.

TL;DR:

- Private lists (for personal sorting)
- Pinned lists (for starred repos tab -- even just a few for key lists like current projects)

**Comment by Sarafs104** [comment]
![f32246c2c7e9c5e2368108f03dc7fec8.png](https://github.com/community/community/assets/168151810/d0add81d-b977-43c7-b4cf-e8f7a76ef080)

**Reply by Toofan777** [reply]
![۲۰۲۴۰۴۲۵_۱۵۵۱۴۰.jpg](https://github.com/community/community/assets/169004335/a813e4c1-3307-4c5f-bcb9-4e28955ca43e)

# ~~~~

**Comment by xingxing789** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> I've been using Lists to sort and categorize repos for myself and others but it would be great imo to have a way to either pin a few of the lists on the $\color{yellow}Starred$ tab. 
> 
> 
> 
> Namely, I have **"Current Projects," "My Projects," and "Hackathons"** as the main lists that I use for others' reference, while the others are just for sorting and personal interest. I've linked them to my Readme but it would be nice to have a setting there too.
> 
> 
> 
> Being able to **private lists** would also be great especially for my misc. lists such as "interesting" which are things I want to look into and read into or "Class" which is just full of private repos hence appearing empty.

**Reply by Nuuux** [reply]
M

**Reply by Nuuux** [reply]
Mane

**Comment by Gopinath-1604** [comment]
To star a list in GitHub, you can curate your starred repositories with public lists. Here's how you can star a list in GitHub:

1. Create Public Lists: You can create public lists that will appear on your stars page at `https://github.com/USERNAME?tab=stars`.
   
2. Add Repositories to Lists: Whenever you see a repository's **_Star_** or **_Starred_** dropdown menu, whether on a repository page or in search results, you can add the repository to an existing list or create a new list.

By following these steps, you can effectively organize your starred repositories into lists on GitHub, making it easier to categorize and manage your repositories.

**Comment by Carlojay17** [comment]
Enable my account owner access support

**Reply by Carlojay17** [reply]
Support to claim account

**Comment by Ridvan8899** [comment]
Jh

**Comment by dockter34** [comment]
Kill

**Reply by dockter34** [reply]
Killl

**Reply by dockter34** [reply]
> Killl

**Reply by Ridvan8899** [reply]
[]()

**Reply by Aslan11021983** [reply]
> []()

😁

**Comment by Aslan11021983** [comment]


**Reply by Aslan11021983** [reply]
122

**Comment by Alisongod** [comment]
> ![f32246c2c7e9c5e2368108f03dc7fec8.png](https://github.com/community/community/assets/168151810/d0add81d-b977-43c7-b4cf-e8f7a76ef080)
> 
>

**Comment by Alisongod** [comment]
> 1
> 8
